test/test_log.py

```diff
@@ -82,6 +82,38 @@ def test_default_logging(
         calls = backend.calls(clear=True)
         assert calls == [result]
 
+    @pytest.mark.parametrize(
+        'message,result',
+        [
+            ('critical', ('CRITICAL', 'critical')),
+            ('error', ('ERROR', 'error')),
+            ('warning', ('WARNING', 'warning')),
+            ('info', ('INFO', 'info')),
+            ('debug', ('DEBUG', 'debug')),
+        ],
+    )
+    def test_verbose_logging(
+        self,
+        backend: FakeModelBackend,
+        logger: logging.Logger,
+        message: str,
+        result: tuple[str, str],
+    ):
+        ops.log.setup_root_logging(backend)
+        ops.log.JujuLogHandler.set_formatting(ops.log.JujuLogHandler.VERBOSE_FORMATTING)
+        assert logger.level == logging.DEBUG
+        assert isinstance(logger.handlers[-1], ops.log.JujuLogHandler)
+
+        method = getattr(logger, message)
+        method(message)
+        calls = backend.calls(clear=True)
+        level, message = result
+        expected = re.compile(rf'{logger.name}:\d+\s+{message}')
```

Comment:
While I like the test, I would prefer a test vector.
Like the exact log line produced with default and verbose formats.

If that's somehow hard (e.g. time), then I'd like to see examples in a comment.